client/src/providers/HealthProvider.tsx

```diff
@@ -1,41 +1,43 @@
-import { ethers } from 'ethers';
-import React, { useState } from 'react';
+import { ethers } from "ethers";
+import React, { useState } from "react";
 
-import { healthBlockAddress } from '../config/constants';
+import { healthBlockABI, healthBlockAddress } from "../config/constants";
 import Web3Modal from 'web3modal';
 import { HealthBlock__factory } from '../contracts/factories/HealthBlock__factory';
 import { HealthBlock } from '../contracts/HealthBlock';
 
-export type Patient = {
-  name: string;
-  age: number;
-  email: string;
-};
+interface Request {
+    doctor: string;
+    doctorName: string;
+    credentialsHash: string;
+    approved: boolean;
```

Comment:
if request.approved is false, there is no distinction between pending and rejected requests. this seems to be a bug. how will the request be associated with health care provider?